apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx

```diff
@@ -198,28 +200,58 @@ class BoardNotFound extends Component {
       <Overlay {...this.props}>
         <Icon icon="exclamation-triangle"/>
         <Text>Make sure your board is plugged in.</Text>
-        <OverlayButton
-          primary
-          text="Try Again"
-          className="try-again"
-          onClick={this.props.handleTryAgain}
-        />
-        <OverlayButton
-          text="Run Without Board"
-          className="run-without-board"
-          onClick={this.handleRunWithoutBoard}
-        />
-        <OverlayButton
-          text="Setup Instructions"
-          className="setup-instructions"
-          onClick={this.props.handleOpenSetupPage}
-        />
+        <UniformWidth>
+          <OverlayButton
+            primary
+            text="Try Again"
+            className="try-again"
+            onClick={this.props.handleTryAgain}
+          />
+          <OverlayButton
+            text="Run Without Board"
+            className="run-without-board"
+            onClick={this.handleRunWithoutBoard}
+          />
+          <OverlayButton
+            text="Setup Instructions"
+            className="setup-instructions"
+            onClick={this.props.handleOpenSetupPage}
+          />
+        </UniformWidth>
       </Overlay>
     );
   }
 }
 
 /**
+ * Wraps a group of buttons in such a way that they will all be as wide
+ * as the widest button.
+ */
+function UniformWidth({children}) {
+  return (
+      <div
+        style={{
+          display: 'flex',
+          flexDirection:'row',
```

Comment:
It does indeed! [CanIUse reports full support](http://caniuse.com/#search=flex) with some IE11 caveats, so I verified on IE11/Win7:

![screenshot from 2017-05-30 10-33-47](https://cloud.githubusercontent.com/assets/1615761/26596482/aea50bd6-4523-11e7-9f0d-6c62d7c8232a.png)